### PR TITLE
Update video core popup

### PR DESCRIPTION
**Pull request by goldenx86**
Old version had formatting issues, and I want to provide an answer to the most common reason this pops up in the first place, outdated drivers.

### CODE REVIEW

File: src/yuzu/main.cpp [hunk 1]
```diff
@@ -1301,13 +1301,12 @@ bool GMainWindow::LoadROM(const QString& filename, u64 program_id, std::size_t p
         case Core::SystemResultStatus::ErrorVideoCore:
             QMessageBox::critical(
                 this, tr("An error occurred initializing the video core."),
-                tr("yuzu has encountered an error while running the video core, please see the "
-                   "log for more details."
+                tr("yuzu has encountered an error while running the video core, 
+                   "this is usually caused by outdated GPU drivers, intcluding integrated ones. "
```
Comment (german77) [inline src/yuzu/main.cpp:1305]: including

File: src/yuzu/main.cpp [hunk 2]
```diff
@@ -1301,16 +1301,13 @@ bool GMainWindow::LoadROM(const QString& filename, u64 program_id, std::size_t p
         case Core::SystemResultStatus::ErrorVideoCore:
             QMessageBox::critical(
                 this, tr("An error occurred initializing the video core."),
-                tr("yuzu has encountered an error while running the video core, please see the "
-                   "log for more details."
+                tr("yuzu has encountered an error while running the video core, "
+                   "this is usually caused by outdated GPU drivers, including integrated ones. "
```
Comment (CrusadingNinja) [inline src/yuzu/main.cpp:1305]:
```suggestion
                tr("Yuzu has encountered an error while running the video core. "
                   "This is usually caused by outdated GPU drivers, including integrated ones. "
```

Comment (CrusadingNinja) [reply src/yuzu/main.cpp:1305]: Grammar errors: Fixed lack of capitalization and comma splice.

**Review by CaptV0rt3x — COMMENTED** [review]
Please replace the forum link with this link

https://yuzu-emu.org/help/reference/log-files/

File: src/yuzu/main.cpp (hunk 2 above)
Comment (CaptV0rt3x) [reply src/yuzu/main.cpp:1305]: @CrusadingNinja 'yuzu' is stylized like that on purpose.

**Review by CaptV0rt3x — APPROVED** [review]
LGTM

**Review by bunnei — APPROVED** [review]

**Pull request merged**